### DISCUSSION TITLE
questions-olympiad-konkur

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#3] [Announcements]
# https://aprd.ir/questions-olympiad-konkur/

مجموعه سوالاتی که دوستان درباره کنکور و المپیاد از من پرسیدند...

https://aprd.ir/questions-olympiad-konkur/

**Comment by pourmand1376** [comment]
این کامنت از سایت Graphcomment - که قبلا استفاده می‌کردم - استخراج شده است. 

**نویسنده کامنت:** «matinkarmi»
**متن:**
سلام وقت بخیر  
من دانشجوی ترم یک مهندسی کامپیوتر هستم بنظرتون زورمو روی المپیاد بزارم خوبه یا اینکه صرفا دروس دانشگاهی رو خوب بخونم و وقت اضافی رو روی یادگیری مهارت های برنامه نویسی و فیلد مورد علاقم بزارم؟؟؟

**Reply by pourmand1376** [reply]
سلام متین جان.  
مسئله صنعت یا آکادمیک مسئله مهم و قابل توجهی هست. راستش هنوز انقدر سازمان یافته بهش فکر نکردم که بتونم در این مورد مطلب مفیدی بنویسم ولی یادداشت کردم اگر زمانی تونستم قدری سازمان یافته‌تر بهش فکر کنم در موردش می‌نویسم.

**Comment by codesamira** [comment]
سلام .ممنونم بابت اشتراک گذاری این اطلاعات من بار شاید سومه ک این نوشترو می خونم و هربار بیشتر از دفعه ی قبل نکات میگیرم و انجام میدم ولذت می برم

**Reply by pourmand1376** [reply]
خوشحالم که براتون مفید بوده.

**Comment by Zahra-Mazinani** [comment]
سلام ، ممنون از اینکه تجربتون رو توضیح دادید. خیلی کمک کننده هست. میخواستم بپرسم برای درس شبکه برای المپیاد برای حل مساله از چه کتابی خوب هست کمک بگیریم ؟ اینکه فقط کوروس و راس رو بخونیم کافیه؟

**Reply by pourmand1376** [reply]
سلام زهرا. 

راستش مدت‌ها از زمانی که من المپیاد دادم گذشته. ممکن هست که کتاب جدیدی اومده باشه ولی من ازش بی‌اطلاع باشم. 

با این حال مهم‌ترین مرجع من همین کتاب کراس راس بود. مسائل آخر فصل رو هم فراموش نکن. اونجا نکاتی رو می‌بینی که طراحان سؤال بهش توجه ویژه دارند. 

در [این سایت](http://gaia.cs.umass.edu/kurose_ross/online_lectures.htm) هم سوالات و آزمایش‌هایی که خود طراحان کتاب (به عنوان متریال آنلاین) در نظر گرفتند، هست. برای من مفید بود.

**Comment by Zahra-Mazinani** [comment]
خیلی متشکرم🙏🌱

**Reply by aida-79** [reply]
متاسفانه من خیلی دیر بفکر المپیاد افتادم و از الان که چیزی حدود ۶۰ روز تا ازمون مرحله اول المپیاد مونده تازه میخوام شروع کنم به خوندن ایا ارزش وقت گذاشتن داره و امیدی به موفقیت داشته باشم تو این زمینه یا وقت مو براش نگذارم ؟ سوال دوم اگر قرار باشه بخونم ایا کار درستیه که درس ساختمان و الگوریتم و حذف کنم ؟ پایه ام بنظرم تو درس نظریه خوبه ولی تو بقیه دروس نه و باید حتما وقت بزارم و بخونم با توجه به زمان کم چه پیشنهادی میدید ؟

**Comment by aida-79** [comment]
متاسفانه من خیلی دیر بفکر المپیاد افتادم و از الان که چیزی حدود ۶۰ روز تا ازمون مرحله اول المپیاد مونده تازه میخوام شروع کنم به خوندن ایا ارزش وقت گذاشتن داره و امیدی به موفقیت داشته باشم تو این زمینه یا وقت مو براش نگذارم ؟ سوال دوم اگر قرار باشه بخونم ایا کار درستیه که درس ساختمان و الگوریتم و حذف کنم ؟ پایه ام بنظرم تو درس نظریه خوبه ولی تو بقیه دروس نه و باید حتما وقت بزارم و بخونم با توجه به زمان کم چه پیشنهادی میدید ؟
